data/neo4j/neo4j_database.py

```diff
@@ -4,7 +4,7 @@
 query_periodic_commit = "USING PERIODIC COMMIT 10000 "
```

Comment:
If I leave this file named neo4j it thinks its a circular import for some reason.